rules/macos/credential_access_promt_for_pwd_via_osascript.toml

```diff
@@ -55,8 +55,15 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where host.os.type == "macos" and event.type in ("start", "process_started") and process.name : "osascript" and
- process.command_line : "osascript*display dialog*password*"
+process where event.action == "exec" and
+ process.name : "osascript" and process.args : "-e" and process.command_line : ("*osascript*display*dialog*password*", "*osascript*display*dialog*passphrase*") and
+ not (process.parent.executable : "/usr/bin/sudo" and process.command_line : "*Encryption Key Escrow*") and
+ not (process.command_line : "*-e with timeout of 3600 seconds*" and user.id == 0 and process.parent.executable : "/bin/bash") and
```

Comment:
I believe the unit test is complaining about user.id.

IIRC it should be 'user.id == "0"'